sdks/java/testing/tpcds/src/main/java/org/apache/beam/sdk/tpcds/TpcdsParametersReader.java

```diff
@@ -17,17 +17,25 @@
  */
 package org.apache.beam.sdk.tpcds;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 /** Get and check the TpcdsOptions' parameters, throw exceptions when user input is invalid. */
 public class TpcdsParametersReader {
+
   /** The data sizes that have been supported. */
   private static final Set<String> supportedDataSizes =
       Stream.of("1G", "10G", "100G").collect(Collectors.toCollection(HashSet::new));
 
+  private static final String QUERY_PREFIX = "query";
+
+  public static final List<String> SUPPORTED_QUERIES = getSupportedQueries();
```

Comment:
I am wondering if we should rename this one. It gives the impression that Beam SQL supports all queries which is not the case, maybe we just call it `TPC_DS_QUERIES` or `QUERIES`.